include/ck/tensor_operation/operator_transform/transform_conv_fwd_to_gemm.hpp

```diff
@@ -14,6 +14,17 @@
 namespace ck {
 namespace tensor_operation {
 
+// function to be used on device, emulates std::accumulate
+template <typename T, typename ForwardIterator, typename Size>
+__host__ __device__ auto mult_accumulate_n(ForwardIterator first, Size count, T init)
```

Comment:
Maybe reuse accumulate_n from library/include/ck/library/utility/numeric.hpp ?

Reply:
The version in CK uses std::accumulate, which I believe is not supported for device code.